assets/javascripts/modules/autoComplete.js

```diff
@@ -277,14 +284,20 @@ var isMatchingSuggestion = function (value) {
   if (value) {
     $(suggestions).each(function (index, suggestion) {
       if (value.toLowerCase() === suggestion.title.toLowerCase()) {
-        updateInput(suggestion.title, suggestion.value);
-        matchFound = true;
-        return false;
+        if ($autoCompleteInputElem.data('suggestionValue').toString() === suggestion.value) {
+          matchFound = true;
+          return false;
```

Comment:
needs to return false within the two conditionals, as the (non-stated) else allows for the multiple variants that we are trying to handle, and should not return